lessons/Data-Science/data-collection/langs/en-us/pages/bad-survey.adoc

```diff
@@ -1,8 +1,8 @@
 = Bad Questions Make Dirty Data
 
-The @starter-file{bad-survey} has __lots__ of problems, which can lead to many kinds of dirty data: Missing Data, Inconsistent Types, Inconsistent Units and Inconsistent Language! Try filling it out with bad data. Record the problems and make some recommendations for how to improve the survey!
+The Height v Wingspan Survey has __lots__ of problems, which can lead to many kinds of dirty data: Missing Data, Inconsistent Types, Inconsistent Units and Inconsistent Language! Try filling it out with bad data. Record the problems and make some recommendations for how to improve the survey!
```

Comment:
I notice you \took out the use of `@starter-file` here. I assume that's because we want kids to be working on a _copy_ that the teacher made, right? In that case, we should tell the student "Your teacher has made a copy of this survey for your class, and will give you a link to their copy." it's an extra check against teachers forgetting.

Also, why no `[.linkInstructions]`?